### PR TITLE
Time Changed for Daylight Savings Time

### PR DESCRIPTION
**Pull request by Eshah7**
## Trello ticket link

[Ticket Name](https://trello.com/b/RCwqUZD8/msci-342-scrum-board)

## Implementation description
* The time was updated, please ensure that the correct time/date is displayed for My Events, Events, My Comments, Comments, Forums and My Forums. 
* Also, please ensure the date filters are working well. 

## Steps to test
1.

## What should reviewers focus on?
*

### CODE REVIEW

**Review by safchow — CHANGES_REQUESTED** [review]
`Dashboard.js` needs to be updated

**Review by c45choi — APPROVED** [review]

File: src/components/events/Events.js [hunk 1]
```diff
@@ -292,7 +292,7 @@ const Events = () => {
               </Text>
 
               <Text className="body to-text" marginTop="8px">
-                Date: {new Date(new Date(event.date).getTime() - (5 * 60 * 60 * 1000)).toLocaleString()}
+                Date: {new Date(new Date(event.date).getTime() - (4 * 60 * 60 * 1000)).toLocaleString()}
```
Comment (safchow) [inline src/components/events/Events.js]: just realized this: for events, the timestamp itself (ex: 8:00PM) isn't important. Can you update the `toLocaleString()` method to `toLocaleDateString()`? You need to do the same for My Events

File: src/components/events/Events.js [hunk 2]
```diff
@@ -363,7 +363,7 @@ const Events = () => {
               className="body to-text"
               marginTop="2px"
             >
-              {new Date(new Date(selectedEvent.date).getTime() - (5 * 60 * 60 * 1000)).toLocaleString()}
+              {new Date(new Date(selectedEvent.date).getTime() - (4 * 60 * 60 * 1000)).toLocaleString()}
```
Comment (safchow) [inline src/components/events/Events.js]: refer to previous comment

File: src/components/events/MyEvents.js [hunk 3]
```diff
@@ -466,7 +466,7 @@ const MyEvents = () => {
                             </Text>
 
                             <Text className="body to-text" marginTop="8px">
-                                Date: {new Date(new Date(event.date).getTime() - (5 * 60 * 60 * 1000)).toLocaleString()}
+                                Date: {new Date(new Date(event.date).getTime() - (4 * 60 * 60 * 1000)).toLocaleString()}
```
Comment (safchow) [inline src/components/events/MyEvents.js]: refer to previous comment

**Review by safchow — CHANGES_REQUESTED** [review]
remove the timestamp from events

**Review by safchow — APPROVED** [review]

**Pull request merged**